week-02/assessment/gomoku/src/main/java/learn/gomoku/GameController.java

```diff
@@ -0,0 +1,128 @@
+package learn.gomoku;
+
+import learn.gomoku.game.Gomoku;
+import learn.gomoku.game.Result;
+import learn.gomoku.game.Stone;
+import learn.gomoku.players.HumanPlayer;
+import learn.gomoku.players.Player;
+import learn.gomoku.players.RandomPlayer;
+
+import java.util.List;
+import java.util.Scanner;
+
+import static learn.gomoku.game.Gomoku.WIDTH;
+
+public class GameController {
+
+    private Scanner console = new Scanner(System.in);
+    Gomoku game;
+
+
+    public void run(){
+        setup();
+        play();
+        playAgain();
```

Comment:
MAJOR: you call playAgain() here, but there's no loop so it just returns a boolean and then the game always ends